book/src/integrations/oauth2.md

```diff
@@ -286,25 +295,61 @@ kanidm system oauth2 disable-localhost-redirects <name>
 kanidm system oauth2 enable-localhost-redirects mywebapp
 ```
 
-## Alternate Redirect Origins
+## Alternate Redirect URLs
 
 Some services may have a website URL as well as native applications. These native applications
 require alternate redirection URLs to be configured so that after an OAuth2 exchange, the system can
 redirect to the native application.
 
-To support this Kanidm allows supplemental origins to be configured on clients.
+To support this Kanidm allows supplemental opaque origins to be configured on clients.
 
-{{#template ../templates/kani-warning.md imagepath=../images title=WARNING text=The ability to configure multiple origins is NOT intended to allow you to share a single Kanidm client definition between multiple OAuth2 clients. This fundamentally breaks the OAuth2 security model and is NOT SUPPORTED as a configuration. Multiple origins is only to allow supplemental redirects within the _same_ client application. }}
+{{#template ../templates/kani-warning.md imagepath=../images title=WARNING text=The ability to
```

Comment:
Can't line wrap these 'cause it breaks the template, have to put the deno exception tags in ... found somewhere else